src/components/journeys/JourneyInstancesDataTable/index.tsx

```diff
@@ -43,29 +46,40 @@ const JourneyInstancesDataTable: FunctionComponent<JourneysDataTableProps> = ({
 
   return (
     <>
-      <UserConfigurableDataGrid
+      <DataGridPro
         checkboxSelection
         columns={columnsWithHeaderTitles}
         components={{ Toolbar: Toolbar }}
         componentsProps={{
           toolbar: {
             gridColumns: columnsWithHeaderTitles,
             setQuickSearch,
-            setSortModel,
-            sortModel,
+            setSortModel: modelGridProps.onSortModelChange,
```

Comment:
I would have taken this opportunity to rename `setSortModel` (which I'm no fan of, because it leaks state changing) to `onSortModelChange`, which I think is a better naming scheme for a callback which communicates up the component tree.

Reply:
LOVE this, this further clarifies the tree of responsibilities! Will note this and use in the future where I come across similar situations.